include/revng/Pipeline/Loader.h

```diff
@@ -194,6 +199,7 @@ struct llvm::yaml::MappingTraits<pipeline::StepDeclaration> {
     TheIO.mapOptional("Pipes", Info.Pipes);
     TheIO.mapOptional("EnabledWhen", Info.EnabledWhen);
     TheIO.mapOptional("Artifacts", Info.Artifacts);
+    TheIO.mapOptional("Analysis", Info.Analysis);
```

Comment:
Why do we allow a single analysis per step? I think we want to have more than one.